tests/hardhat/ERC4626/VenusERC4626Core.ts

```diff
@@ -349,18 +349,11 @@ describe("VenusERC4626Core", () => {
     const rewardAmount = ethers.utils.parseEther("10");
 
     describe("When rewardRecipient is EOA", () => {
-      it("should claim rewards and transfer to recipient", async () => {
+      it("should revert the transaction", async () => {
```

Comment:
Is not the try-catch block avoiding the TX to be reverted? Maybe it's being reverted in the test for other reason

Reply:
<img width="876" height="407" alt="Screenshot 2025-07-28 at 1 50 48 PM" src="https://github.com/user-attachments/assets/74f186bc-a16b-4e2a-bc1c-7c07d8251ac3" />
 
 link : https://rareskills.io/post/try-catch-solidity#part-2-how-try-catch-handle-each-situation